javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,82 @@
+console.log("Hello"); //Initial code
+
+//Age-ify (A future age calculator)
+const yearOfBirth = 1989;
+const yearFuture = 2050;
+
+//with using - arithmatic operator we can calculate the age.
+let myAge  = yearFuture  - yearOfBirth;
+
+console.log("I will be " + myAge + " years old in " + yearFuture);
```

Comment:
Check Template literals and how you can write these statements in a more elegant way https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals